lib/elixir/lib/code/normalizer.ex

```diff
@@ -0,0 +1,246 @@
+defmodule Code.Normalizer do
+  @moduledoc false
+
+  defguard is_literal(x)
+           when is_integer(x) or
+                  is_float(x) or
+                  is_binary(x) or
+                  is_atom(x)
+
+  @doc """
+  Wraps literals in the quoted expression to conform to the AST format expected
+  by the formatter.
+  """
+  def normalize(quoted, opts \\ []) do
+    line = Keyword.get(opts, :line, 1)
+
+    do_normalize(quoted, line: line)
+  end
+
+  # Skip already wrapped blocks
+  defp do_normalize({:__block__, _, [literal]} = quoted, _parent_meta) when is_literal(literal) do
+    quoted
+  end
+
+  defp do_normalize({:__block__, meta, args} = quoted, _parent_meta) do
+    if Keyword.has_key?(meta, :format) do
```

Comment:
Why do we skip in this case?

Reply:
It's a leftover from a previous iteration, it can be safely removed